lib/embulk/command/embulk_migrate_plugin.rb

```diff
@@ -43,8 +43,8 @@ def self.migrate_java_plugin(migrator, from_ver)
     end
 
     # upgrade gradle version
-    if migrator.match("gradle/wrapper/gradle-wrapper.properties", /gradle-2\.\d-/)
-      # gradle < 2.10 (\d matches one digit)
+    if migrator.match("gradle/wrapper/gradle-wrapper.properties", /gradle-2\.\d+-/)
```

Comment:
👍 Note.  
Next time, we will need to care the gradle version 2.X and < 3.2.1. 
ie) `/gradle-[23]\.\d+(\.\d+)?-/` (not tested yet).